README.md

```diff
@@ -116,8 +116,526 @@ Run workflows on the web server, passing `data`, `flux`, and `fix`:
 
 ## Module
 
-The repo contains and uses a new `Metafix` stream module for Metafacture which plays the role of the `Metamorph` module in Fix-based Metafacture workflows. For the current implementation of the `Metafix` stream module see `MetafixDslTest.java`. For a real-world usage sample see [https://gitlab.com/oersi/oersi-etl/-/blob/develop/data/production/edu-sharing.fix](https://gitlab.com/oersi/oersi-etl/-/blob/develop/data/production/edu-sharing.fix).
+The repo contains and uses a new `Metafix` stream module for Metafacture which plays the role of the `Metamorph` module in Fix-based Metafacture workflows. For the current implementation of the `Metafix` stream module see `MetafixDslTest.java`. For a real-world usage sample see [https://gitlab.com/oersi/oersi-etl/-/blob/master/data/production/openRub/openRub.fix](https://gitlab.com/oersi/oersi-etl/-/blob/master/data/production/openRub/openRub.fix).
 
 # Xtext
 
 This repo has been originally set up with Xtext 2.17.0 and Eclipse for Java 2019-03, following [https://www.eclipse.org/Xtext/documentation/104_jvmdomainmodel.html](https://www.eclipse.org/Xtext/documentation/104_jvmdomainmodel.html).
+
+
+## Functions and Cookbook
+
+### Best practices and guidelines for working with Metafacture Fix
+
+- We recommend to use double quotation marks for arguments and values in functions, binds and conditionals.
+- If using a `list` bind with a variable, the `var` option requires quotation marks (`do list(path: "<sourceField>", "var": "<variableName>")`).
+- Fix turns repeated fields into arrays internally but only marked arrays (with `[]` at the end of the field name) are also emitted as "arrays" (entities with indexed literals), all other arrays are emitted as repeated fields.
+- Every Fix file should end with a final newline.
```

Comment:
While 1.) and 4.) are recommendations/best practices, 2.) and 3.) are kind of hints about how Metafix works. It might make sense to have different lists for these in the future. For now it is ok I guess.